### PR TITLE
Run conftest test and reorder a little

### PR DESCRIPTION
**Pull request by krajorama**
Signed-off-by: György Krajcsovits <gyorgy.krajcsovits@grafana.com>

#### Which issue(s) this PR fixes or relates to

Fixes #<issue number>

#### Checklist

- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/internal/contributing/contributing-to-helm-chart.md [hunk 1]
```diff
@@ -113,3 +113,9 @@ This is useful to limit otherwise noisy output to only show objects of a certain
 cd operations/compare-helm-with-jsonnet
 ./compare-kustomize-outputs.sh ./helm/09-* ./jsonnet/09-* 'select(.kind == "StatefulSet")'
 ```
+
+### Static checks
+
+Use the make targets `check-conftest-fmt`, `conftest-verify` and `conftest-test` to lint, verify and execute [conftest](https://www.conftest.dev/) static analysis tests.
```
Comment (dimitarvdimitrov) [inline docs/internal/contributing/contributing-to-helm-chart.md:119]:
```suggestion
Use the make targets `conftest-fmt`, `conftest-verify` and `conftest-test` to lint, verify and execute [conftest](https://www.conftest.dev/) static analysis tests.
```

File: docs/internal/contributing/contributing-to-helm-chart.md [hunk 2]
```diff
@@ -113,3 +113,9 @@ This is useful to limit otherwise noisy output to only show objects of a certain
...
+
+The tests are verifying that the policies defined in `operations/helm/policies` are met for all Kubernetes manifests generated from configurations defined under `operations/helm/charts/mimir-distributed/ci` and `operations/helm/charts/mimir-distributed/ci/conftest` directories.
```
Comment (dimitarvdimitrov) [inline docs/internal/contributing/contributing-to-helm-chart.md:121]: nit: one is a subdir of the other
```suggestion
The tests are verifying that the policies defined in `operations/helm/policies` are met for all Kubernetes manifests generated from configurations defined under `operations/helm/charts/mimir-distributed/ci`.
```

**Review by dimitarvdimitrov — APPROVED** [review]
lgtm, some nitpicks

File: docs/internal/contributing/contributing-to-helm-chart.md (hunk 2 above)
Comment (krajorama) [reply docs/internal/contributing/contributing-to-helm-chart.md:121]: just wanted to be specific, but I see your point :)

**Pull request merged**